### ISSUE TITLE
Suggestion: Use Google Maps Javascript SDK by default

### ISSUE THREAD

**Issue by joshuamorony** [driftyco/ionic-starter-super#65]
I think the current implementation of the maps page is a bit confusing. 

By default, it will use the JS SDK in the browser, but switch to using the native plugin when on a device. Since they are two totally different things I think this could confuse people who are using this starter as the basis for learning how to implement maps. 

In my opinion, the JS SDK should be preferred by default for both the browser and cordova, perhaps with a commented out function to initialise the native maps. I feel like mixing the native SDK with the web view is wrought with issues, and shouldn't be the default.

Any thoughts?

**Comment by mlynch** [comment]
Good point, configuring this also fits w/ a broader config system for these kinds of tradeoffs. Let me think about the default plugin and how to make this easier...

**Comment by MikeLallemont** [comment]
Anybody have a current implementation of google maps with Ionic 3.3.0? I'll take the Javascript implementation.

**Comment by janpio** [comment]
The current one here doesn't work?

**Comment by MikeLallemont** [comment]
I set up and verified the APIs and I get a map ready message on the console but the screen is blank.
